src/disco/components/Addon/index.js

```diff
@@ -242,6 +244,30 @@ export class AddonBase extends React.Component {
     }
   };
 
+  installStaticTheme = async (event) => {
+    event.preventDefault();
+    const {
+      enable,
+      hasAddonManager,
+      isAddonEnabled,
+      install,
+      status,
+    } = this.props;
+
+    if (hasAddonManager && status === UNINSTALLED) {
```

Comment:
do we really need to check the addonManager?

Reply:
i thought it would good to ck to be sure. We do pass this val into the button so thought there could be scenario would it's false for some reason here too, no?